app/controllers/links_controller.rb

```diff
@@ -29,15 +29,21 @@ def external_url(type:, options: {})
 
   private
 
+  def foreman_org_path(sub_path)
+    "https://theforeman.org/#{sub_path}"
+  end
+
   def documentation_url(section = "", options = {})
-    root_url = options[:root_url] || "https://theforeman.org/manuals/#{SETTINGS[:version].short}/index.html#"
+    root_url = options[:root_url] || foreman_org_path("manuals/#{SETTINGS[:version].short}/index.html#")
     root_url + (section || '')
   end
 
-  def plugin_documentation_url(plugin_name, version: nil, options: {})
-    root_url = options[:root_url] || "https://theforeman.org/plugins"
+  def plugin_documentation_url(options)
+    root_url = options[:root_url] || foreman_org_path("/plugins")
+    plugin_name = options[:name]
+    version = options[:version]
     path = version ? "#{plugin_name}/#{version}" : plugin_name
-    "#{root_url}/#{path}"
+    "#{root_url}/#{path}#{options[:section]}"
```

Comment:
is there a better way to destruct the options hash?
in ES6 there is `const { name, version, section } = options`

maybe passing it like that: `plugin_documentation_url(*options)` ?

Reply:
I suppose you can use [`values_at`](https://apidock.com/ruby/Hash/values_at):

``` ruby


hash = {:a => 1, :b => 2, :c => 3}
c, d = hash.values_at(:a, :b)
c # => 1
d # => 2
```